src/transfer/configuration_loader.py

```diff
@@ -2,24 +2,16 @@
 import argparse
 import os
 import re
-from enum import Enum
 from pathlib import Path
 
 import yaml
 from numcodecs import Blosc
+from transfer.readers import EphysReaders
 
 
 class EphysJobConfigurationLoader:
     """Class to handle loading ephys job configs"""
 
-    class RegexPatterns(Enum):
-        """Enum for regex patterns the source folder name should match"""
-
-        subject_datetime = r"\d+_\d{4}-\d{2}-\d{2}_\d{2}-\d{2}-\d{2}"
-        ecephys_subject_datetime = (
-            r"ecephys_\d+_\d{4}-\d{2}-\d{2}_\d{2}-\d{2}-\d{2}"
-        )
-
     def __remove_none(self, data):
```

Comment:
I think it would be good if we add typings for parameters in methods and functions. Makes the code more legible.